src/NativeUi.elm

```diff
@@ -111,19 +111,14 @@ unsafeRenderDecodedProperty =
 
 {-| -}
 style : List Style.Style -> Property msg
-style styles =
-    Style.encode styles
-        |> Native.NativeUi.style
+style =
+    Native.NativeUi.style << Style.encode
 
 
 {-| -}
 on : String -> Decoder msg -> Property msg
-on eventName =
-    let
-        realEventName =
-            "on" ++ eventName
-    in
-        Native.NativeUi.on realEventName
+on =
+    Native.NativeUi.on << ((++) "on")
```

Comment:
Sure, although I just realized that that might unnecessarily pollute the global scope. Might actually be better to not go point-free here and revert to a similar solution as was already here?

```elm
{-| -}
on : String -> Decoder msg -> Property msg
on eventName =
    let
        realEventName =
            if String.startsWith "on" eventName then
                eventName
            else
                "on" ++ eventName
    in
        Native.NativeUi.on << getEventName
```